osf/models/user.py

```diff
@@ -1805,22 +1808,25 @@ def get_or_create_cookie(self, secret=None):
         :returns: The signed cookie
         """
         secret = secret or settings.SECRET_KEY
```

Comment:
A comment for myself: this might be an existing bug ... `SECRET_KEY` is defined in `websites/settings/defaults.py` and should be referred as `website_settings` here. Not sure why nothing is broken.

```
[from django.conf import settings](https://github.com/CenterForOpenScience/osf.io/blob/b5c28d2b096c0090fc34f7f2ad151fb9d546e333/osf/models/user.py#L19)
...
[from website import settings as website_settings](https://github.com/CenterForOpenScience/osf.io/blob/b5c28d2b096c0090fc34f7f2ad151fb9d546e333/osf/models/user.py#L55)
```

Reply:
There is also a `SECRET_KEY` in django settings. It simplys imports the `SECRET_KEY` from `website.settings`.
https://github.com/CenterForOpenScience/osf.io/blob/develop/api/base/settings/defaults.py#L47

Reply:
As discussed, `from django.conf import settings` is the same as `from api.base import settings`.

> In https://github.com/django/django/blob/5659015/django/conf/__init__.py#L63, it loads whatever settings that is referred by `DJANGO_SETTINGS_MODULE`